exp/api/v1alpha4/azuremanagedcontrolplane_types.go

```diff
@@ -100,6 +104,33 @@ type AADProfile struct {
 	AdminGroupObjectIDs []string `json:"adminGroupObjectIDs"`
 }
 
+// LoadBalancerProfile - Profile of the cluster load balancer.
+type LoadBalancerProfile struct {
+	// ManagedOutboundIPs - Desired managed outbound IPs for the cluster load balancer.
+	// +optional
+	ManagedOutboundIPs *int32 `json:"managedOutboundIPs,omitempty"`
+
+	// OutboundIPPrefixes - Desired outbound IP Prefix resources for the cluster load balancer.
+	// +optional
+	OutboundIPPrefixes []string `json:"outboundIPPrefixes,omitempty"`
+
+	// OutboundIPs - Desired outbound IP resources for the cluster load balancer.
+	// +optional
+	OutboundIPs []string `json:"outboundIPs,omitempty"`
+
+	// EffectiveOutboundIPs - The effective outbound IP resources of the cluster load balancer.
+	// +optional
+	EffectiveOutboundIPs []string `json:"effectiveOutboundIPs,omitempty"`
```

Comment:
are you familiar with the impact of sending this value? 

I believe it's a ready only field which is the union of the resource IDs from OutboundIPs and the resource IDs of the AKS-created ManagedOutboundIPs, and thus should not be in our spec.

Reply:
Thanks for pointing it out! I checked the [api doc](https://docs.microsoft.com/en-us/rest/api/aks/managed-clusters/create-or-update) again and it's indeed only in the api response, i'll remove it from the spec.